### DISCUSSION TITLE
What is the logic behind making breakpoints in em by default?

### DISCUSSION THREAD

**Discussion by skorphil** [chakra-ui/chakra-ui#8342] [General]
Hi, why default breakpoints are in em rather than px?

https://chakra-ui.com/docs/styled-system/responsive-styles

**Comment by segunadebayo** [comment]
Default breakpoints use `em` so they track the user's root font size. Zoom and browser text scaling stay in sync. `px` would ignore that.

Override `theme.breakpoints` if you need pixel values. https://chakra-ui.com/docs/styling/responsive
